src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6182,4 +6178,8 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
     <comment>Do not localize project.json and `PackageReference` 
 0 - project path (project.json file path)</comment>
   </data>
+  <data name="Error_UnknownDependencyVersion" xml:space="preserve">
```

Comment:
I wouldn't rename the name of the resource. 
It'd break localization without more changes.

Reply:
I can switch back to  **InstallCommandUnknownDependencyVersion** but, wouldn't it be strange to have a resource name called "Installxxx" even if this resource is used for "Update" behavior.

Furthermore I didn't found any localization for this resource (usually the resource seems localized with " ResourceName_Language"  but i didn't find any of them for  "InstallCommandUnknownDependencyVersion".
Do the localization are contained in another project?

Reply:
Nope, they are not, they usually in the same file. 
You're right this wouldn't affect localization so renaming it is ok. 
Nice catch :)